src/molecules/formfields/shared/formfields.module.scss

```diff
@@ -163,8 +167,31 @@ $checkbox-border-color: $neutral-5;
   }
 }
 
-// Select Field - types
 
+//
+.datefield {
+  composes: formfield;
```

Comment:
Agree. We should avoid using composes in the future. Our sass should be able to compile without using CSS Loader.